lbz/dev/server.py

```diff
@@ -22,6 +24,11 @@ class MyLambdaDevHandler(BaseHTTPRequestHandler):
     done = False
 
     def _get_route_params(self, org_path):
+        """
+        Parses route and params.
+        :param org_path:
+        :return: standarised route, url params / None
```

Comment:
I would take advantege of typing library here to avoid comments:
_get_route_params(self, org_path: str) -> Union[tuple(?), None]